### ISSUE TITLE
Memcache: SERVER_ERROR Object too large for cache in logs

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#12934]
Logs for "Object too large for cache" over past 7 days:

STAGING: https://vagov.ddog-gov.com/logs?query=service%3Acms-drupal%20deployment_name%3Acms%20environment%3Astaging%20Object%20too%20large%20for%20cache&cols=service&index=&messageDisplay=inline&stream_sort=time%2Cdesc&viz=stream&from_ts=1677889677061&to_ts=1678494477061&live=true

![image](https://user-images.githubusercontent.com/1504756/224453609-959a28f0-da68-45a3-9101-d9e7c6785f10.png)

PROD: https://vagov.ddog-gov.com/logs?query=service%3Acms-drupal%20deployment_name%3Acms%20environment%3Aprod%20Object%20too%20large%20for%20cache&cols=service&index=&messageDisplay=inline&stream_sort=time%2Cdesc&viz=stream&from_ts=1677889677061&to_ts=1678494477061&live=true (241 results)

![image](https://user-images.githubusercontent.com/1504756/224453663-98b07fae-da17-4a1d-abc4-a927b24eb8e4.png)

This appears like it may be resolveable with the memcached (or Elasticache Memcache) config setting of `max_item_size`, I set this from `1M` to `4M` already on a new param group I made on STAGING and the last result is from Mar 6th, around the time I set it. This fix for this may just be confirming that and doing some tests and confirming that is a good number. We will probably want to set this on Tugboat and DDEV too. 

## Acceptance Criteria
- [x] Determine if issue is solved
- [x] Put config change in code
- [x] Apply new config to STAGING, TEST-STAGING
- [ ] Apply new config to PROD, TEST-PROD

**Comment by ElijahLynn** [comment]
I applied a fix for this on March 8th and I'm seeing this error hasn't been in our logs since March 6th, so it does appear that it worked. I'm working on another ticket still but wanted to leave this here for now: 

https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/gh/department-of-veterans-affairs/va.gov-cms/12745#issuecomment-1461078556

![image.png](https://images.zenhubusercontent.com/5cdc72c50b7efe3f164150b2/1e5bdffa-695b-47c7-b9e0-903e9ca90ada)

**Comment by ElijahLynn** [comment]
I've pulled the config changes into code and tested the Terraform by applying to test-staging and staging. Next step is to merge these changes and apply to test-prod and prod. I believe we don't have to have an outage window for this, as if the CMS doesn't find a memcache server it uses the DB cache. 

I was able to load the site while it was rebooting the nodes, so this appears to be verified. 

![image](https://user-images.githubusercontent.com/1504756/227817778-69feee86-e3ad-47e7-97cd-bfcf9252dfa6.png)


Here are the two PRs to merge for the PROD deploy, which I'll do tomorrow during business hours:

https://github.com/department-of-veterans-affairs/terraform-aws-vsp-cms/pull/39
https://github.com/department-of-veterans-affairs/devops/pull/12726
